Implement audio and video action

PrP-11 opened the pull request:
Fixes #1218 

Changes: Implemented audio and video actions

Demo Link: http://pricey-distribution.surge.sh/

Screenshots for the change: 
![api](https://user-images.githubusercontent.com/30981465/40360705-9c2c9372-5de4-11e8-992d-a36d301841fb.png)

mariobehling commented:
@PrP-11 Please implement this in the way that it automatically plays the media item.

mariobehling commented:
Please showcase this with the command "play" as well.

PrP-11 commented:
@mariobehling we need to create a new skill to showcase this with the play command and since skills cms in not working currently so I am unable to create the skill. The example above is a test case which @Orbiter has appended in susi skills [here](https://github.com/fossasia/susi_skill_data/blob/cf9f801d122fbd1b1d4a85db824d0a3120682b53/models/general/Games%2C%20Trivia%20and%20Accessories/en/hal_persona.txt). I have already created a skill for that on dream [here](https://dream.susi.ai/p/youtube). I also created an issue for that [here](https://github.com/fossasia/susi_skill_cms/issues/469). The work of client is to implement the action type only.

PrP-11 commented:
@mariobehling first I implemented video to start automatically but then a problem will arise, if we reload the page, all the previous songs will play automatically. I have implemented automatic feature [here](http://lovely-window.surge.sh/), please review it whether it is fine with you.

duttaditya18 commented on src/components/ChatApp/MessageListItem/MessageListItem.react.js:
```diff
@@ -248,6 +248,45 @@ class MessageListItem extends React.Component {
+            case 'video_play': {
+              let identifierType = data.answers[0].actions[index].identifier_type;
+              let identifier = data.answers[0].actions[index].identifier;
+              let src = `https://www.${identifierType}.com/embed/${identifier}`;
```
You can do something like : 
```javascript
let src = `https://www.${identifierType}.com/embed/${identifier}?autoplay=1`;
```

This should work in most popular Video Hosting Websites.

PrP-11 replied:
@duttaditya18 please see older comments first and give your views accordingly. Do you have any solution for the above bug? Thanks. :-)

rishiraj824 replied:
@PrP-11 I think you can identify if you are fetching from the History API or not. If you are, then using a certain configuration for the `src` attribute to not add the autoplay query will solve the case.

isuruAb commented:
It gave me an output like this. 
<img width="415" alt="screen shot 2018-05-23 at 10 57 15 pm" src="https://user-images.githubusercontent.com/7692626/40440815-b4aefbe4-5edc-11e8-8688-06aff61c8c77.png">

duttaditya18 commented:
@isuruAb I am getting the same output.

mariobehling commented:
@PrP-11 Please implement autoplay and try the suggestion @duttaditya18 

We need a solution that actually "plays" the video/sound, if the user says "play". This is the goal of the feature to -> play.

> The Query String ?autoplay=1 works on YouTube and on some other players as well.

mariobehling commented:
@PrP-11 You can also research on timelines of Facebook, Youtube and Linkedin. They also embed videos that autoplay if they are on-screen. Please go into details to make this possible.

PrP-11 commented:
@mariobehling I have already implemented autoplay and updated the PR. You can check it here http://cooperative-bedroom.surge.sh/

AnupKumarPanwar commented:
@PrP-11 In my case, autoplay is not working in this link . http://cooperative-bedroom.surge.sh/. Also there the chat goes out of order. 
Browser used - Chrome
